### ISSUE TITLE
Import an Audio Work (metadata only) from Wax Cylinder MARC File

### ISSUE THREAD

**Issue by mark-dce** [curationexperts/alexandria-v2#551]
Given one or more sample MARC records, I would like to be able to create an Audio Work for the corresponding Wax Cylinder recordings.

- [ ] Import the Title
- [ ] Import the Performer (or personal names)
- [ ] Import the System ID (pegasus record number)
- [ ] Import the Type of Work (work_type) 
- [ ] If you re-import an existing MARC record (i.e. the system number matches), all metadata should be overwritten by the new MARC record, i.e. overwrite existing data & make sure that any fields not in the MARC get cleared if they had previous values set.

We'll write another story for all the remaining fields, but this is sufficient to unblock other related import stories.

Assume that the MARC recordset to import will be stored in a directory on the local server.

**Comment by mark-dce** [comment]
The following pegasus query will give you MARCxml records matching all of the sample wave files we have:
http://pegasus-test.library.ucsb.edu:5661/sba01pub?version=1.1&operation=searchRetrieve&maximumRecords=20&query=rec.id=(4544615%20or%204110610%20or%203633067%20or%203240608%20or%203245508%20or%203240764%20or%202855754%20or%202600681%20or%202556253%20or%202559341)

I've saved the output on the server under /data/metadata-cylinders/cylinders_samples_only_marc.xml

**Comment by jcoyne** [comment]
I don't see ARKs for these in the MARC records.  Do the ARKs exist for these or are we meant to mint them?

**Comment by ilessing** [comment]
ARKs for Cylinders will be created outside ADRL.  This is similar to ETDs.  So the importer does not need to mint arks.  If you need ARKs in the records for testing/development purposes we should dummy some up.

**Comment by jcoyne** [comment]
@ilessing that is good to know. Will the ARKs be added to the MARC records?  Currently I'm not seeing ARKs in the MARC.

**Comment by ilessing** [comment]
Yes, the plan is to add the ARK to the MARC record in Aleph.  When that will happen I can not say.

**Comment by jcoyne** [comment]
@ilessing I'm not sure we can go forward with ingesting the wax cylinders until they have been given ARKs (or we give them test ARKs)

**Comment by mark-dce** [comment]
@ilessing to clarify, we will need ARKs assigned in the MARC records before UCSB can do a production ingest.  We can complete development with test ARKs.
